### PR TITLE
 task_5939_create_ui_mini_music

### PR DESCRIPTION
**Pull request by QNFC**
Create UI mi music
Link redmine: https://edu-redmine.framgia.vn/issues/5939

### CODE REVIEW

**Comment by QNFC** [comment]
ready

File: app/src/main/res/layout/mini_music_controller.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/navigator_background"
+    >
+
+    <ImageView
+        android:id="@+id/imageViewPosterMini"
+        android:layout_width="72dp"
```
Comment (phuongohn-1626) [inline app/src/main/res/layout/mini_music_controller.xml]: hard code

File: app/src/main/res/layout/mini_music_controller.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        android:layout_height="50dp"
```
Comment (phuongohn-1626) [inline app/src/main/res/layout/mini_music_controller.xml]: hard code

**Comment by QNFC** [comment]
ready

File: app/src/main/java/com/framgia/music_48/screen/main/MainActivity.java [hunk 3]
```diff
@@ -33,10 +121,100 @@ private void initDrawerLayout() {
...
+                mNavigationView.getMenu().getItem(OptionTab.TAB_HOME).setChecked(true);
+                break;
+            case OptionTab.TAB_GENRES:
+                mNavigationView.getMenu().getItem(OptionTab.TAB_GENRES).setChecked(true);
+                break;
+            case OptionTab.TAB_SEARCH:
+                mNavigationView.getMenu().getItem(OptionTab.TAB_SEARCH).setChecked(true);
+                break;
+            default:
+                mNavigationView.getMenu().getItem(OptionTab.TAB_GENRES).setChecked(true);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_48/screen/main/MainActivity.java:214]: add break

File: app/src/main/java/com/framgia/music_48/service/MusicService.java [hunk 4]
```diff
@@ -81,11 +86,42 @@ public void musicPlay() {
         updateUI(song);
     }
 
+    public boolean checkMediaPlayer() {
+        if (mSongs.size()== 0){
+            return false;
+        }
+        else {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_48/service/MusicService.java]: reformat code

**Comment by QNFC** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**